packages/modules/azureservicebus/src/azureservicebus-container.test.ts

```diff
@@ -0,0 +1,64 @@
+import { ServiceBusClient } from "@azure/service-bus";
+import { getImage } from "../../../testcontainers/src/utils/test-helper";
+import { ServiceBusContainer } from "./azureservicebus-container";
+
+const IMAGE = getImage(__dirname);
+
+describe("ServiceBuscontainer", { timeout: 180_000 }, () => {
+  it("should connect and queue a message", async () => {
+    // serviceBusConnect {
+    await using container = await new ServiceBusContainer(IMAGE).acceptLicense().start();
+    const client = new ServiceBusClient(container.getConnectionString());
+    const sender = client.createSender("queue.1");
+
+    await sender.sendMessages({ body: "Hello, World!" });
```

Comment:
Is it easy enough to test that we can also receive the message? Just to give a nice end-to-end example for the docs. It'd also help verify that the configuration test below works